### ISSUE TITLE
Unable to install yaml. TransportException (HTTPS?)

### ISSUE THREAD

**Issue by amcsi** [FriendsOfPHP/pickle#149]
On fresh master

I'm trying to install yaml, but I'm getting this and I have no idea what I can do about it.

```
$ ~/.composer/vendor/bin/pickle install yaml

  - Installing yaml (latest-stable)



  [Composer\Downloader\TransportException]                                                                                                              
  Your configuration does not allow connections to http://pecl.php.net/get/yaml. See https://getcomposer.org/doc/06-config.md#secure-http for details.  



install [--no-convert] [--defaults] [--dry-run] [--php="..."] [--ini="..."] [--source] [--with-configure-options="..."] [--save-logs="..."] [path]
```

**Comment by pierrejoye** [comment]
yes, https should solve this.

It is coming today

**Comment by pierrejoye** [comment]
fixed in master.

**Comment by pierrejoye** [comment]
fixed in 0.5.0 release
